source/_integrations/solax.markdown

```diff
@@ -36,6 +36,11 @@ port:
   type: integer
   default: 80
   description: The port number
+password:
```

Comment:
We no longer allow changes to the YAML configuration. Please see my more extensive comment on the parent PR for more information.